frontend/src/views/DistanceCharts.vue

```diff
@@ -60,8 +60,9 @@ const chartStore = useChartsStore()
 const hydrologicStore = useHydrologicStore()
 
 // TODO:nodes on tab switch, update distance chart data by query nodes
-let nodeVariables = hydrologicStore.getPlottableSwotVariables('node')
-let varTab = ref(nodeVariables[0])
+//let nodeVariables = hydrologicStore.getPlottableSwotVariables('node')
+let nodeCharts = chartStore.nodeCharts
```

Comment:
this will break reactivity
see: https://pinia.vuejs.org/api/pinia/functions/storeToRefs.html#Function-storeToRefs-

either use chartStore.nodeCharts directly, or use storeToRefs